addon/classes/Table.js

```diff
@@ -192,6 +192,24 @@ export default class Table extends EmberObject.extend({
   }
 
   /**
+   * The same as `setRows`, however the given array is synced, meaning that
+   * mutating the array also updates the table.
+   *
+   * @method setRows
+   * @param  {Array} rows
+   * @param  {Object} options
+   * @return {Array} rows
+   */
+  setRowsSynced(rows = [], options = {}) {
+    let _rows = RowSyncArrayProxy.create({
+      syncArray: rows,
+      content: emberArray(Table.createRows(rows, options))
```

Comment:
slightly tangential question: you wouldn't happen to know if there's any downside to https://github.com/buschtoens/ember-light-table/blob/master/addon/classes/Table.js#L155 doesn't pass in the options to `Table.createRows()`